tests/src/JIT/Performance/CodeQuality/Span/SpanBench.cs

```diff
@@ -412,13 +500,31 @@ public static void TestSpanDangerousCreateByte(int length)
         [InlineData(1000)]
         public static void TestSpanDangerousCreateString(int length)
         {
-            TestClass<string> testClass = new TestClass<string>();
-            testClass.C0 = new string[length];
-            Span<string> span;
-            foreach (var iteration in Benchmark.Iterations)
-                using (iteration.StartMeasurement())
-                    for (int i = 0; i < Benchmark.InnerIterationCount; i++)
-                        span = Span<string>.DangerousCreate(testClass, ref testClass.C0[0], testClass.C0.Length);
+            InvokeTestSpanDangerousCreate<string>(length);
+        }
+
+        static void InvokeTestSpanDangerousCreate<T>(int length)
+        {
+            TestClass<T> testClass = new TestClass<T>();
```

Comment:
`TestSpanDangerousCreateByte` calls `InvokeTestSpanDangerousCreate<byte>` calls `Invoke` calls the lambda call `TestSpanDangerousCreate<byte>` calls `Span<byte>.DangerousCreate`.  Am I missing something?